### PR TITLE
Update to new logo (Modelica Language one to be specific.)

### LINKED ISSUE #2426: Add logos to title/second page and header
The new logos could be added to title/second page and header when creating the PDF, similar as [FMI](https://github.com/modelica/fmi-standard/files/3694391/FunctionalMockupInterface_v2.0.1_RC6.pdf).

### PR DESCRIPTION
**Pull request by HansOlsson**
Adding header (with new logo)
Closes #2426
Note: This requires fancyhdr-package; I didn't figure out what to update to add that to CI-system.

### CODE REVIEW

**Comment by beutlich** [comment]
CI artefacts: [MLS.pdf](https://test.openmodelica.org/jenkins/job/ModelicaAssociation/job/ModelicaSpecification/job/PR-2448/1/artifact/MLS.pdf) and [MLS.tar.gz](https://test.openmodelica.org/jenkins/job/ModelicaAssociation/job/ModelicaSpecification/job/PR-2448/1/artifact/MLS.tar.gz)

File: MLS.tex [hunk 1]
```diff
@@ -10,6 +10,10 @@
 % Add new Modelica Language logotype
 % The header ruler looks odd as Modelica Language define a natural line that is further up
 % We also need to fill the vertical space on the right
+% Not  using page numbers, since we use reference using sections.
```
Comment (svorkoetter) [inline MLS.tex]: This sentence isn't quite right. Do you mean "since we use reference sections", or "we reference using sections" ?

Comment (HansOlsson) [reply MLS.tex]: Hmm... "since we usually reference using sections."?

File: MLS.tex [hunk 2]
```diff
@@ -7,6 +7,17 @@
 \begin{document}
 % Title
 \maketitle
+% Add new Modelica Language logotype
+% The header ruler looks odd as Modelica Language define a natural line that is further up
+% We also need to fill the vertical space on the right
+% Not  using page numbers, since we use reference using sections.
+%
+% Using nouppercase since it is seems more normal for the sections, and is also
```
Comment (beutlich) [inline MLS.tex]: The trailing white space causes the CI build to fail.

File: MLS.tex (hunk 1 above)
Comment (HansOlsson) [reply MLS.tex]: Went with "Not using page numbers in right-header, since we usually refer to sections."

**Comment by HansOlsson** [comment]
@beutlich @svorkoetter are there any remaining issues? If not can you accept-review?

**Pull request merged**